### PR TITLE
Fix often forget zip python package

### PR DESCRIPTION
**Pull request by YayunHuang**
Ref: MUP-164

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -71,6 +72,13 @@ check-format:
 bundle-python:
```
Comment (pkong-ds) [inline Makefile]: can you help change `bundle-python` as `bundle-python-mac` too? thanks!

Comment (YayunHuang) [reply Makefile]: @pkong-ds updated

File: Makefile [hunk 2]
```diff
@@ -67,10 +68,17 @@ check-format:
...
+bundle-python-mac:
 	ditto -c -k --sequesterRsrc --keepParent ./mockup_package/mockup ./public/mockup.zip
 
+.PHONY: bundle-python-linux
+bundle-python-linux:
+	mkdir -p mockup
+	cp -r mockup_package/mockup/* mockup/
+	zip -r public/mockup.zip mockup
+	rm -rf mockup
+
```
Comment (pkong-ds) [inline Makefile]: Sorry, forgot to say - if we are creating&removing sth in a script, can we follow the convention of putting these into `tmp/` folder? thanks

**Review by pkong-ds — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (YayunHuang) [reply Makefile]: @pkong-ds updated~

File: Makefile [hunk 3]
```diff
@@ -67,10 +68,22 @@ check-format:
...
+	mkdir -p tmp/mockup
+	cp -r mockup_package/mockup/ tmp/mockup/
+	cd tmp && \
+	if [ -d "mockup" ]; then \
+	  zip -r ../public/mockup mockup; \
+	else \
+	  echo "Error: Directory 'mockup' does not exist."; \
+	  exit 1; \
+	fi
+
```
Comment (pkong-ds) [inline Makefile]: Why need cd into `tmp`?

```suggestion
	if [ -d "tmp/mockup" ]; then \
	  zip -r public/mockup tmp/mockup; \
	else \
	  echo "Error: Directory 'mockup' does not exist in directory 'tmp'."; \
	  exit 1; \
	fi

```

**Review by pkong-ds — CHANGES_REQUESTED** [review]

File: Makefile (hunk 3 above)
Comment (YayunHuang) [reply Makefile]: @pkong-ds  if we use `zip -r public/mockup tmp/mockup;` the unzip file will be
```
tmp/
    mockup/
```

Comment (pkong-ds) [reply Makefile]: okay, let's keep it then

**Review by pkong-ds — APPROVED** [review]
Added `rm -rf tmp`, @YayunHuang please merge if okay!

**Pull request merged**